index.js

```diff
@@ -0,0 +1,108 @@
+const billInput = document.querySelector(".bill-input");
+const billAlert = document.querySelector(".bill-alert");
+
+const tipRatioInputs = document.querySelectorAll(".select-tip-button");
+const customTipRatioInput = document.querySelector(".custom-tip-input");
+
+const peopleInput = document.querySelector(".people-input");
+const peopleAlert = document.querySelector(".people-alert");
+
+const tipPerPersonOutput = document.querySelector(".tip-per-person");
+const totalPerPersonOutput = document.querySelector(".total-per-person");
+const resetButton = document.querySelector(".reset-button");
+
+let bill = 0;
+let tipRatio = 0;
+let people = 0;
+let tipPerPerson = 0;
+let totalPerPerson = 0;
+
+const calculate = () => {
+  if (bill === 0 || tipRatio === 0 || people === 0) {
+    return;
+  }
+
+  tipPerPerson = (bill * tipRatio) / people;
+  totalPerPerson = bill / people + tipPerPerson;
+
+  tipPerPersonOutput.innerHTML = `$ ${tipPerPerson.toFixed(2)}`;
+  totalPerPersonOutput.innerHTML = `$ ${totalPerPerson.toFixed(2)}`;
+};
+
+const onBillChange = (e) => {
+  if (e.target.value === "0") {
+    billAlert.style.visibility = "visible";
+    e.target.style.border = "1px solid red";
+  } else {
+    billAlert.style.visibility = "hidden";
+    e.target.style.border = "none";
+
+    bill = Number(e.target.value);
```

Comment:
단항 연산자 `+` 를 사용하면 명시적으로 숫자형으로 변환할 수 있습니다.
한번 사용해보시는 것도 좋을 것 같아요!
```suggestion
    bill = +e.target.value;
```

Reply:
사실 당시에 `parseInt()`가 생각나지 않아서 썼던 코드인데 이 방법이 더 간편하네요!

Reply:
오 저도 알아갑니다!!